assets/src/dashboard/app/views/templateDetails/index.js

```diff
@@ -202,6 +205,7 @@ function TemplateDetails() {
             <Layout.Fixed>
               <TemplateNavBar
                 handleCta={() => createStoryFromTemplate(template)}
+                handleBookmarkClick={enableBookmarks}
```

Comment:
For me, it's odd to pass a boolean to a prop that reads as a verb which would tell me a function. For the sake of correctness I would do something like
```
handleBookmarkClick={enableBookmarks ? () => {} : undefined}
```

You could even create an empty

```
const handleBookmarkClick = useCallback(() => {}, [])
```
as a placeholder no-op and pass that as a prop.

Reply:
makes sense!